Runtime/Scripts/RGBotConfigs/DefaultActions/DrawIndicator.cs

```diff
@@ -0,0 +1,66 @@
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace RegressionGames.RGBotConfigs.DefaultActions
+{
+    
+    /**
+     * This action uses Gizmos to draw an indicator at a specific position in a scene.
+     * Takes the following parameters:
+     *  If modifying an indicator (either new or old)
+     *    - name (string): And identifier for the indicator
+     *    - position (Vector3): The position to place the indicator
+     *    - color (Color): The color of the indicator
+     *    - size (float): The size of the indicator
+     *  If removing the indicator:
+     *    - name (string): The indicator to remove
+     *    - remove (boolean): Set to true
+     *  If removing all indicators:
+     *    - removeAll (bool): Set to true
+     */
+    public class DrawIndicator: RGAction
+    {
+
+        private Dictionary<string, Tuple<Vector3, Color, float>> _indicators = new ();
+
+        public override string GetActionName()
+        {
+            return "DrawIndicator";
```

Comment:
Let's name this what it is `DrawSphere` or `DrawWireSphere` as there are many useful indicator types in `Gizmos`

Reply:
Fixed/new code